bower.json

```diff
@@ -21,15 +21,29 @@
   "homepage": "https://github.com/PolymerElements/iron-swipeable-container",
   "ignore": [],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.1.0"
+    "polymer": "Polymer/polymer#^2.0.0"
```

Comment:
In order for this element to work in "hybrid" mode, please use an inclusive version range that allows for 1.x compatible dependencies. Example: https://github.com/PolymerElements/paper-button/blob/master/bower.json#L24-L27